tests/test_io.py

```diff
@@ -137,4 +223,187 @@ def test_no_records(self, no_records: pd.DataFrame) -> None:
                 "Encounter_number",
             ],
         )
-        pd.testing.assert_frame_equal(no_records, expected)
+
+        pd.testing.assert_frame_equal(result, expected)
+
+    def test_both_breaks_repaired(self, sample_file: SampleFileFixture) -> None:
+        result = sample_file(
+            path="tests/data/both_breaks.txt",
+            on_broken_records="repair",
+        )
+        expected = pd.DataFrame(
+            {
+                "EMPI": ["012345", "1515156"],
+                "EPIC_PMRN": ["012345", "33445567"],
+                "MRN_Type": ["MGH", "BWH"],
+                "MRN": ["012345", "444444"],
+                "Report_Number": ["999999", "0123456"],
+                "Report_Date_Time": ["1/1/2000 9:00:00 AM", "1/1/2012 8:00:00 PM"],
+                "Report_Description": ["ABC\r\nDE", "ABCDE"],
+                "Report_Status": ["F", "F"],
+                "Report_Type": ["GHIJK", "GHIJK"],
+                "Report_Text": [
+                    (
+                        "\r\nThis is an example report text. "
+                        "The format may not reflect a typical note."
+                        "\r\n\r\nPatient name: First Last\r\n\r\n"
+                        "Sex: Male\r\n\r\nDOB: 1/1/1950\r\n\r\n"
+                        "HPI: ?\r\n[report_end]"
```

Comment:
My preference would be to strip `[report_end]` since I see it as part of the RPDR format and not part of the text.

Reply:
Updated in most recent commit(s).